services/static-webserver/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM joseluisq/static-web-server:1.15.0-alpine as base
+
+LABEL maintainer=neagu@itis.swiss
+
+# front-end client last (image name is the path to the Dockerfile)
+COPY --from=client/tools/qooxdoo-kit/builder:latest --chown=scu:scu \
+    /project/build-output "/static-content"
```

Comment:
user scu exists in static-web-server:1.15.0? from the [Dockerfile](https://github.com/joseluisq/static-web-server/blob/master/docker/alpine/Dockerfile) it seems to run with defaults, meaning root user.
for enhanced security you should probably try to change the user when running that docker instead of running as root.

Reply:
you are correct. I've just missed it.

Reply:
Ok so I just tried it now locally and here is what I can show you:
1. ```make build-x up-prod```
2. find the id of the static-webserver docker container
3. ```docker exec -it ID sh```
4. ```apk add procps```
5. ```ps -aux```

you should see this:
```
/ # ps -aux
USER         PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root           1  0.0  0.1 808988 52812 ?        Ssl  22:01   0:00 static-web-server
root         206  0.0  0.0   1656  1132 pts/0    Ss   22:02   0:00 sh
root         223  0.0  0.0   1632   864 pts/0    R+   22:05   0:00 ps -aux
```

so basically the container is still run by the root user.

to compare this is how it looks like in the webserver:
```
USER         PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root           1  0.0  0.0   1104     0 ?        Ss   22:01   0:00 /sbin/docker-init -- services/web/server/docker/en
scu            7  3.8  0.4 9081600 136340 ?      Sl   22:01   0:08 /home/scu/.venv/bin/python /home/scu/.venv/bin/sim
root         142  0.0  0.0   2384   760 pts/0    Ss   22:03   0:00 sh
root         151  0.0  0.0   5748  3648 pts/0    S    22:03   0:00 bash
root        1352  0.0  0.0  12896  9044 ?        Rs   22:04   0:00 python3 /home/scu/services/web/server/docker/healt
root        1358  0.0  0.0   9388  3048 pts/0    R+   22:04   0:00 ps -aux
```

you can see that the scu user is running the server

Reply:
For now I'm disabling non root user until https://github.com/joseluisq/static-web-server/issues/42 is addressed or I get a reply to the issue.